src/components/discussion-link.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Discussion Link
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const DiscussionLink = ({linkType, link}) => {
+  if (link && link != "") {
+    return (
+      <a
```

Comment:
I think this a great simple solution for this for new. Maybe in future, if these links are working well, then we can upgrade them a bit, like have it `Discuss on: HackerNews, LinkedIn, Reddit, etc.`.

Reply:
Yeah I was hoping to be able to create a map in the meta data but it was not so easy so I did it the simple way first. 

Issue added: https://github.com/datavaluepeople/dvp-landing-static/issues/68